generic/tests/cfg/boot_vm_in_hugepage.cfg

```diff
@@ -10,8 +10,8 @@
     pre_command_noncritical = yes
     pre_command = "echo 3 > /proc/sys/vm/drop_caches"
     variants:
-        - 2M_hugepage:
-            expected_hugepage_size = 2048
+        - @default:
```

Comment:
1. not all arch's default hugepage is 2048.
2. if the host has set 1G hugepage already, then /proc/meminfo is 1G, I think the previous code if better.